Cogs/StudentCommands.py

```diff
@@ -58,56 +59,44 @@ async def helloworld(self, ctx, language='random'):
-                   'c': '```c\n#include <stdio.h>\n\nint main() {\n    printf("Hello world!\\n");\n    return 0;\n}```',
-                   'bash': '```bash\necho "Hello world!"```',
-                   'javascript': '```javascript\nconsole.log("Hello world!");```',
-                   'brainf': '```\n++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.>++.```',
-                   'rust': '```rust\nfn main() {\n    println!("Hello World!");\n}```',
-                   'matlab': '```matlab\ndisp(\'hello world\')```',
-                   'html': '```html\n<!DOCTYPE html>\n\n<html>\n  <head>\n    <title>Hello world!</title>\n    <meta charset="utf-8" />\n  </head>\n\n  <body>\n    <p>Wait a minute. This isn\'t a programming language!</p>\n  </body>\n</html>```',
-                   'csharp': '```csharp\nnamespace CSEBot {\n    class HelloWorld {\n        static void Main(string[] args) {\n            System.Console.WriteLine("Hello World!");\n        }\n    }\n}```',
-                   'vb': '```vb\nImports System\n\nModule Module1\n    Sub Main()\n        Console.WriteLine("Hello World!")\n        Console.WriteLine("Press Enter Key to Exit.")\n        Console.ReadLine()\n    End Sub\nEnd Module```',
-                   'r': '```r\nprint("Hello World!", quote = FALSE)```',
-                   'go': '```go\npackage main\n\nimport "fmt"\n\nfunc main() {\n    fmt.Println("Hello world!")\n}```',
-                   'swift': '```swift\nimport Swift\nprint("Hello world!")```',
-                   'haskell': '```haskell\nmodule Main where\nmain = putStrLn "Hello World"```',
-                   'befunge': '```befunge\n64+"!dlrow olleH">:#,_@```',
-                   'perl': '```perl\nprint "Hello world!"```',
-                   'php': '```php\n<?php\necho \'Hello World\';\n?>```',
-                   'lisp': '```lisp\n(DEFUN hello ()\n  (PRINT (LIST \'HELLO \'WORLD))\n)\n(hello)```',
-                   'basic': '```basic\n10 PRINT "Hello World"\n20 END```',
-                   'cobol': '```cobol\n       identification division.\n       program-id. cobol.\n       procedure division.\n       main.\n           display \'Hello world!\' end-display.\n           stop run.```',
-                   'papyrus': '```papyrus\nScriptname helloworld\n\nEvent OnPlayerLoadGame()\n\tDebug.Notification("Hello World!")\nEndEvent```'}
+        # Read in the langague data from the yaml file
+        with open('helloworld.yml', 'r') as f:
+            language_data = yaml.load(f)
+
 
         # List languages
-        if language == 'ls':
-            languages = [i for i in outputs]
+        if language.lower() == 'ls':
+            languages = [i for i in language_data]
             languages.sort()
             languages = '\n'.join(languages)
             await ctx.send(f'I know:\n{languages}')
             return
 
         # If invalid input, make it random
         language = language.lower()
```

Comment:
I think this line should be earlier, since a .lower() was added to the comparison on line 80. Maybe put a comment of "Cleaned Input" followed by language = language.lower()? Up to you, this is a super nitpick.